docs/geedocs/answer/2987876.html

```diff
@@ -17,15 +17,36 @@
 <h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Install GEE Server</h1>
```

Comment:
I'd like to move this somewhere else. It doesn't seem to belong in this section and it is also difficult to find through navigating the menu.

Reply:
I moved the page to its own section under the main menu and removed it from the old section.